src/Pages/List/Components/Drink.js

```diff
@@ -0,0 +1,189 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import Tooltip from './Tooltip';
+
+const Drink = ({ toggleTooltip, value, selectedTooltip, sendToServer }) => {
+  const [drink, setDrink] = useState([]);
+  const [isChecked, setIsChecked] = useState({
+    맥주: false,
+    위스키: false,
+    와인: false,
+    칵테일: false,
+  });
+
+  const changeOption = item => {
+    setIsChecked({ ...isChecked, [item.name]: item.checked });
+    const copyDrink = [...drink];
+    copyDrink.includes(item.name)
+      ? setDrink(copyDrink.filter(drink => drink !== item.name))

```

Comment:
```suggestion
      ? setDrink(drink.filter(drink => drink !== item.name))
```
로 해도 됩니다. drink배열 안의 하나의 아이템이 drink인건 이상하니까 drinks등으로 state이름 수정해주면 좋을 것 같네요

Reply:
수정했어요!